### ISSUE TITLE
Do we need average_gradient when we do mutiprocess distributed training?

### ISSUE THREAD

**Issue by dzk9528** [pytorch/examples#659]
In the tutorial, it is said that we need to write `avereage_gradients` to get the average gradient for different process, then we can do `optimizer.step()`, however, in the imagenet example, `avereage_gradients`  is not there. Does it means we do not need this function in new version of pytorch for mutiprocess distributed training?(I am using torch 1.3.0)

**Comment by soumith** [comment]
you simply adjust the learning rate according to the number of processes, so that you scale the sum down to average
